### ISSUE TITLE
Part 5: Add table of x-ogc- properties

### ISSUE THREAD

**Issue by m-mohr** [opengeospatial/ogcapi-features#945]
I think it would be valuable to add a table of all the pre-defined additions to JSON Schema, i.e. with the `x-ogc-` prefix.
It would help to understand schemas more easily, especially as some schemas consist of more proprietary properties than JSON Schema vocabulary.

PS: Generally, it feels like the number of additions to JSON Schema is a bit excessive.

**Comment by pvretano** [comment]
SWG Meeting 29-JUL-2024:  Yes, all present agreeds that a table of `x-ogc-` values would be useful.  @pvretano will add.

**Comment by jerstlouis** [comment]
In addition to having a table in the Standard, it might be worthwhile to set up some kind of registry so that additional fields can be registered for different use cases, after the Standard has been published. cc. @rob-metalinkage @avillar

**Comment by jerstlouis** [comment]
I've created a Wiki page on the OGC API - Common repo to keep track of the `x-ogc-*` extension keywords until the Naming Authority sets up a register for them:

https://github.com/opengeospatial/ogcapi-common/wiki/Table-of-extensions-to-the-JSON-Schema-vocabulary

I used ASCIIDoc so hopefully this can serve as a starting point to address this issue as well.

cc. @ghobona @rob-metalinkage @joanma747 @fmigneault @m-mohr

**Comment by m-mohr** [comment]
I appreciate the effort. IMHO it underlines my point about the excessiveness of these additions and that it feels more and more like OGC Schema with some JSON Schema rather than JSON Schems with some OGC extensions.

**Comment by jerstlouis** [comment]
@m-mohr I disagree mainly in the sense that the vanilla JSON Schema provides all of the essential functionality (mainly the field names, data type, titles, description), whereas clients aware of the additional OGC-specific extensions can leverage the additional information.
